src/south/SQLDbToFile/SQLDbToFile.class.js

```diff
@@ -134,9 +154,7 @@ class SQLDbToFile extends ProtocolHandler {
 
     if (result.length > 0) {
       result.forEach((entry) => {
-        if (entry[this.timeColumn] && (entry[this.timeColumn] instanceof Date) && (entry[this.timeColumn] > new Date(this.lastCompletedAt))) {
-          this.lastCompletedAt = entry[this.timeColumn].toISOString()
-        }
+        this.checkLastCompletedAt(entry)
```

Comment:
I would call it set LastCompletedAt and maybe it would be clearer to wrtie:
this.lastCompletedAt = this.setLastCompletedAt(entry)